Code/Framework/AzFramework/AzFramework/Windowing/NativeWindow.cpp

```diff
@@ -90,82 +97,170 @@ namespace AzFramework
 
     bool NativeWindow::IsActive() const
     {
-        return m_pimpl != nullptr && m_pimpl->IsActive();
+        if (m_pimpl != nullptr)
+        {
+            return m_pimpl->IsActive();
+        }
+        else
+        {
+            return false;
+        }
```

Comment:
The if else blocks aren't needed here.
A conditional expression using `&&` can be used.

This is the @lsemp3d pet-peeve special as the `return false/return true` statement is never needed in this case.
```suggestion
        return m_pimpl != nullptr && m_pimpl->IsActive();
```